### ISSUE TITLE
IB Earthshatter

### ISSUE THREAD

**Issue by funnytroll** [WarEmu/WarBugs#14028]
Skill dont do damage only snare
https://imgur.com/Ct0LpUJ

**Comment by Panzarpaddan** [comment]
I second this on my IB aswell.

**Comment by Skullcapzone** [comment]
Hi devs, we got a patch today which fixed the buffs/class stuff dissapearing, but IB's earthshatter broke at the same time, its snare component works but no matter what level of grudge you have stored up it doesn't do damage on any of the stages its supposed to do damage on.

I have checked at the 25 grudge, 50 grudge, 75 grudge and 100 grudge levels, at 100 grudge its supposed to do 500 damage for all targets effected in 30 distance for my character. On target dummies, npcs and players only the snare seems to happen and no damage, combat log also doesn't seem to report anything for earthshatter either.

Could this possibly get patched for next weekend as its a good part of the IB's AoE rotation and makes PQ grinding quite abit slower.
